Packs/AppID_PAC/Scripts/BuildWebAppID/BuildWebAppID.yml

```diff
@@ -0,0 +1,17 @@
+commonfields:
+  id: BuildWebAppID
+  version: -1
+dockerimage: jinja
```

Comment:
The docker image is not from demisto docker image, you can choose either use an existing demisto docker image or please add your docker image to that repo.
When the docker image is ready, please rename it to: demisto/<image>:<tag>
You can find some explanation regarding docker image: https://xsoar.pan.dev/docs/integrations/docker